### ISSUE TITLE
Thanks a lot!

### ISSUE THREAD

**Issue by pbek** [aurelien-rainone/macdeployqtfix#5]
Thanks a lot for this great tool that makes it possible for me to use `macdeployqt` on current versions of Qt with brew on Travis CI for <https://github.com/pbek/QOwnNotes>!

And a big thank you to @iltommi for fixing the problems with brew!

This was bugging me for ages in QOwnNotes! `macdeployqt` is still broken in Qt 5.8, but the app works like a charm after running `macdeployqtfix`! (Qt 5.5.1 on brew was the last version where `macdeployqt` worked.)

**Comment by arl** [comment]
Hello.

Happy to hear that. :D

Le 2 févr. 2017 09:42, "Patrizio Bekerle" <notifications@github.com> a
écrit :

**Comment by Zverik** [comment]
I agree, this is an awesome tool. By the way, that macdeployqt issue will be fixed in 5.10, due in November.

**Comment by arl** [comment]
Thankx for the good news. about time!

**Comment by muammar** [comment]
Sorry to give more noise, but thank you very much for this!. Very helpful script!. 👍

**Comment by MartinDelille** [comment]
@arl It looks like Qt 5.12 doesn't have these issue with Homebrew installed package anymore. Is macdeployqtfix still usefull for some case?

By the way I would have loved to discover your project before!

**Comment by MartinDelille** [comment]
Ok sorry for the noise: I just read @Zverik comment about 5.10 fix. Great news!
